streams-contrib/streams-provider-twitter/src/main/jsonschema/org/apache/streams/twitter/api/FollowingIdsRequest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://json-schema.org/draft-03/schema",
+  "$license": [
+    "http://www.apache.org/licenses/LICENSE-2.0"
+  ],
+  "id": "#",
+  "javaType" : "org.apache.streams.twitter.api.FollowingIdsRequest",
+  "javaInterfaces": ["java.io.Serializable"],
+  "properties": {
+    "id": {
+      "description": "The ID of the user for whom to return results for.",
+      "required": false,
+      "type": "integer"
+    },
+    "screen_name": {
+      "description": "The screen name of the user for whom to return results for.",
+      "required": false,
+      "type": "string"
+    },
+    "curser": {
```

Comment:
nice catch. that was wrong a bunch of places.  all fixed now and i watch the providers page as expected.